data/bookkeeping/block.go

```diff
@@ -121,6 +121,21 @@ type (
 		// transactions have ever been committed (since TxnCounter
 		// started being supported).
 		TxnCounter uint64 `codec:"tc"`
+
+		// CompactCertVoters is the root of a Merkle tree containing the
+		// online accounts that will help sign a compact certificate.  The
+		// Merkle root, and the compact certificate, happen on blocks that
+		// are a multiple of ConsensusParams.CompactCertRounds.
```

Comment:
could you please add a note that on rounds that aren't multiple of ConsensusParams.CompactCertRounds, the `CompactCertVoters` would contains a zeroed out digest ?

Reply:
Good idea, thanks!